performance_manager/startup.py

```diff
@@ -45,6 +45,8 @@ def load_environment() -> None:
                 logging.info("setting %s to %s", key, value)
                 os.environ[key] = value
 
+    except FileNotFoundError as fnfe:
+        logging.warning("unable to find env file %s", fnfe)
```

Comment:
don't throw if we can't find the file, just warn that the env hasn't been updated.